File: room-sample/src/main/java/org/codecop/rogue/template/Hello.java
package org.codecop.rogue.template;

import jakarta.inject.Singleton;

@Singleton
public class Hello {

    private String name = "World!";

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

}

File: Three/app/Hello.php
<?php

namespace App;

class Hello {

    private $name;

    public function __construct() {
        $this->name = "World!";
    }

    public function getName(): String {
        return $this->name;
    }

    public function setName(String $name): void {
        $this->name = $name;
    }

}
